test/run-e2e-ocp.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 set -e -u -o pipefail
 
-trap cleanup INT
+trap cleanup EXIT
```

Comment:
Trap only EXIT, if I interrupt run not during `run-e2e.sh`, `cleanup` in `run-e2e-ocp.sh` will not be called.

Reply:
It will be called on `exit` and INT will result in `exit`